keycloak/examples/README.md

```diff
@@ -0,0 +1,52 @@
+# Docker-compose example
+
+This example makes available an enviroment with Keycloak integrated with Kafka.
+
+To run this example, type:
+```
+docker-compose up
+```
+
+To explore Keycloak GUI (graphical user interface) access http://localhost:8080 and login in with user ```admin``` and password ```admin```
```

Comment:
Suggestion:
_[...] and login in with [...]_ => _and log in with_

Reply:
Thanks for the observation, it was fixed.